src/Xamarin.Android.Build.Tasks/Utilities/MonoAndroidHelper.cs

```diff
@@ -830,5 +830,22 @@ public static AndroidRuntime ParseAndroidRuntime (string androidRuntime)
 			// Default runtime is MonoVM
 			return AndroidRuntime.MonoVM;
 		}
+
+		/// <summary>
+		/// Returns true if the input file is newer than the output file.
+		/// </summary>
+		public static bool IsFileOutOfDate (string input, string output)
+		{
+			if (!File.Exists (output))
+				return true;
+
+			var inputInfo = new FileInfo (input);
+			var outputInfo = new FileInfo (output);
```

Comment:
If you keep the logic in C#, you might use [File.GetLastWriteTime](https://learn.microsoft.com/en-us/dotnet/api/system.io.file.getlastwritetime?view=net-9.0) instead, so it doesn't create `FileInfo` objects.

Reply:
This utility method looks handy, should it go in dotnet/android-tools in the [Files.cs ](https://github.com/dotnet/android-tools/blob/main/src/Microsoft.Android.Build.BaseTasks/Files.cs#L19)?